pkg/compactor/compactor_test.go

```diff
@@ -1900,6 +1900,7 @@ func TestOwnUser(t *testing.T) {
 	}
 
 	for name, tc := range testCases {
+		tc := tc
```

Comment:
if the tests called in `t.Run` run in parallel, wouldn't this lead to issues if the value in `tc` gets overwritten by the next iteration of the loop on `testCases`?